### DISCUSSION TITLE
Datepicker: Why is there no Close button?

### DISCUSSION THREAD

**Discussion by rho66** [primefaces/community#290] [PrimeFaces] [answered]
The deprecated Calendar component has the buttons 'Current Date' and 'Close' in the button panel. 
The recommended Datepicker component has the buttons 'Today' and 'Clear' in the button panel.

How can I get a 'Close' button in Datepicker?

**Comment by melloware** [comment, marked as answer]
PrimeTek felt it a redundant button when ESC key or clicking anywhere outside the datepicker closes it so no need for a Close button.  PrimeVue, PrimeReact, PrimeNG all have these same component so they made it consistent.
